internal/driver/glfw/window_notwindows.go

```diff
@@ -1,6 +1,16 @@
+//go:build !windows
```

Comment:
Let's not add new Go features while we still target 1.14

Reply:
I think we should be able to add all of these in one go using `gofmt` later, which probably makes more sense. However, I don’t think they should be causing any issues if we have both.